fixubuntu.sh

```diff
@@ -8,16 +8,16 @@ V=$(/usr/bin/lsb_release -rs)
 # The privacy problems started with v12.10, so earlier versions should do nothing
 
 # Added check because 14.04.1 isn't a number
-if [ $V == "14.04.1" ]; then
+if [ "$V" == "14.04.1" ]; then
```

Comment:
Why not just change `[` and `]` to `[[` and `]]`? It's a bash script, look at the shebang.